src/todo-list/todo-list.js

```diff
@@ -1,28 +1,129 @@
+import { Message } from "../config/message.js";
+import Config from "../config/config.js";
+import { summaryComponent } from "../summary/summary.js";
+
 export class ToDoItem {
-  constructor(task) {
+  constructor(task, id) {
     this.task = task;
     this.isDone = false;
+    this.id = id;
   }
 }
 
 export class ToDoList {
-  static ToDoListDom = document.querySelector(".list");
-
+  #toDoListComponent = new ToDoListComponent();
+  #chatComponent = document.querySelector(".chat-content");
+  #taskArray = []
+  #summaryComponent = summaryComponent;
+  
   constructor() {
-    this.taskArray = [];
     this.isFullyCompleted = false;
+    this.#addSubmitListener();
+    this.#addToDoListComponentListener();
+  }
+
+  #addSubmitListener() {
+    const submitButton = document.getElementById("submit-todo");
+    submitButton.addEventListener("click", () => {
+      const text = document.getElementById("new-todo-item").value;
+      this.#addNewItem(text);
+      document.getElementById("new-todo-item").value = "";
+    });
+
+    const submitInput = document.getElementById("new-todo-item");
+    submitInput.addEventListener("keypress", function(event) {
```

Comment:
If you use `<input type="submit">` within a form, you do not need to add this event manually. 😅